src/Ajax.js

```diff
@@ -63,7 +63,11 @@ class Ajax {
 				resolve(request);
 			};
 			request.onerror = function() {
-				var error = new Error('Request error');
+				var message = 'Request error';
+				if (4 === request.readyState && !request.responseURL) {
```

Comment:
IE doesn't set responseURL after a 302 redirect so this case can be understood as CORS error. Make sure to test it.